docs/developer/developer-setup.md

```diff
@@ -161,23 +161,19 @@ All projects in TruBudget (blockchain, api, frontend, etc) have a docker-compose
 Do **NOT** use other additional ways to set environment variables like inline env-variables or env-variables defined by 'export'. Why? - Because these env-variables will overwrite each which makes it very hard to find mistakes.
```

Comment:
Consider using danger/alert instead of note. 
Insert a extra line break before admonitions

Reply:
done